openedx/openedx-platform
# fix: do not reinitialize the MakoService in the MakoDescriptorSystem

Without this, Studio raises the `mako.exceptions.TemplateLookupException:` with the `Can't locate template for uri 'discussion/_discussion_inline_studio.html'` message.

## Testing instructions

1. In Studio, add a Randomized Content Block and select a library.
2. Add a Discussion XBlock to the same unit.
3. Refresh the page and see that it's rendering correctly.
